lstchain/io/io.py

```diff
@@ -1125,16 +1241,21 @@ def merge_dl2_runs(data_tag, runs, columns_to_read=None, n_process=4):
     """
     from functools import partial
     from glob import glob
-    filepath_glob = glob(f'/fefs/aswg/data/real/DL2/*/{data_tag}/*') # Current format of LST data path
+
+    filepath_glob = glob(
+        f"/fefs/aswg/data/real/DL2/*/{data_tag}/*"
+    )  # Current format of LST data path
```

Comment:
Should this path be not hardcoded so you can actually use it anywhere?

Reply:
This was used in [PR#509](https://github.com/cta-observatory/cta-lstchain/pull/509) I have not used these functions yet.

Reply:
Right. Disregard this comment then. Can be addressed in another PR. Also the calculation of the observation time should be done using #566.